### ISSUE TITLE
[Performance] Accuracy dropped heavily using onnxruntime to inference a model quantized by QAT

### ISSUE THREAD

**Issue by MomoP32** [microsoft/onnxruntime#19850]
### Describe the issue

I use pytorch-quantization to do QAT for a pointpillar model, it works fine during pytorch training, however, when I export the torch model to onnx, accuracy degrades badly. In my opinion, this process should have no loss of accuracy at all.

### To reproduce

ensorRT Version:
8.5.2
NVIDIA GPU:
RTX3060
NVIDIA Driver Version:
470.161.03
CUDA Version:
11.3
CUDNN Version:
8.6

Operating System:
Ubuntu 20.04
Python Version (if applicable):
3.8
PyTorch Version (if applicable):
1..11
pytorch-quantization:
2.1.2

### Platform

Linux

### OS Version

20.04

### ONNX Runtime Installation

Released Package

### ONNX Runtime Version or Commit ID

1.14.1

### ONNX Runtime API

Python

### Architecture

X64

### Execution Provider

Default CPU

### Is this a quantized model?

Yes
